src/model/AssetsCompiler.ts

```diff
@@ -94,6 +95,8 @@ export class AssetsCompiler {
             }
         }
 
+        new NPMResolver(mainRepoPath, this.runConfig.watchFiles, this.runConfig.clean);
```

Comment:
Keep the resolver in a variable and do a proper "shutdown" -> call that in the `AssetsCompiler.shutdown()` method to clean up. Explicitly call the initial "resolve" here - the intiial resolve should block until node_modules are fully available (otherwise tsc may crash since it's not available)